src/useCases/ReportManager.java

```diff
@@ -22,43 +20,45 @@ public class ReportManager {
 
     /**
      * Initializes the report manager.
+     *
      * @param database Database - collection of all entity databases in the program.
      */
-    public ReportManager(Database database){
+    public ReportManager(Database database) {
         this.reportDatabase = database.getReportDatabase();
     }
 
     /**
      * Delete a report from the report database.
+     *
      * @param reportData ReportData - data associated with the report to be deleted.
-     * @return boolean - true if the id associated with the reportData exists in the database. false otherwise.
      */
-    public boolean deleteReport(ReportData reportData){
-        return reportDatabase.remove(reportData.getReportId());
+    public void deleteReport(ReportData reportData) {
+        reportDatabase.remove(reportData.getReportId());
     }
```

Comment:
Why did you remove the boolean return here? Could you explain this design decision?

Reply:
I did not remove that, I believe Ryan made that change after.